src/main/java/pw/itr0/kaba/encrypt/PBE.java

```diff
@@ -118,6 +122,25 @@ private static Cipher getCipher(int mode, char[] password, byte[] salt, int iter
         }
     }
 
+    private static byte[] generateIV(char[] password, byte[] salt) {
+        try {
+            byte[] bytes = new byte[password.length];
+            for (int i = 0; i < password.length; i++) {
+                bytes[i] = (byte) password[i];
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Found 'DD'-anomaly for variable 'bytes' (lines '129'-'129').](https://www.codacy.com/app/ryotan/kaba-vault/file/2655946537/issues/source?bid=3298589&fileBranchId=3327309#l129)